storage/scanner_test.go

```diff
@@ -252,6 +252,42 @@ func TestScannerTiming(t *testing.T) {
 	}
 }
 
+// errorNotCloseTo logs an error when the actual value is not close to the expected value
+func logErrorWhenNotCloseTo(expected, actual time.Duration) {
+	delta := 1 * time.Millisecond
+	if actual < expected-delta || actual > expected+delta {
+		log.Errorf("Expected duration %s, got %s", expected, actual)
```

Comment:
For tests you want to use `testing.T.Errorf` so that the test actually fails. Rather than make this a separate function, I'd probably make it a function pointer defined inside of `TestScannerPaceInterval`:

```
  errorWhenNotCloseTo := func(expected, actual time.Duration) {
    delta := 1 * time.Millisecond
    if actual < expected-delta || actual > expected+delta {
      t.Errorf("expected duration %s, got %s", expected, actual)
    }
  }
```